submissions/ik-web/memory-pair-game/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>memory-pair-game</title>
+    <link rel="icon" href="img/favicon.ico" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Redressed&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    
+    <div class="wrapper">
+
+        <div class="end-game off">
+            <div class="end-game__inner">
+                
+                <div class="end-game__text">
+                    <p>Victory</p>
+                    <p>Congratulations!</p>
+                </div>
+                
+                <button class="end-game__btn" id="btn">Play again</button>
+                <div class="end-game__img"><img src="img/sunduk.png" alt="treasure"></div>
```

Comment:
don't use names like this one - `sunduk` =)

Reply:
I changed name for this image.